Fix #56 - Folder name issue on windows

brahma19 opened the pull request:
https://github.com/iamkroot/ilc-scraper/issues/56

iamkroot commented on ilc_scrape.py:
```diff
@@ -11,6 +11,7 @@
 import requests
 from downloader import DirServer, download_stream
 from utils import print_quit, read_json, sanitize_filepath, sp_args, store_json
+from pathvalidate import sanitize_filename
```
The package needs to be added to dependency file using poetry

brahma19 replied:
Added package to poetry dependency file. 
Bumped the version as well, is that needed ?

iamkroot commented on ilc_scrape.py:
```diff
@@ -273,7 +274,7 @@ def main():
 
     total_lecs = len(lectures)
-    subject_name = "{subjectName} {sessionName}".format(**lectures[0])
+    subject_name = sanitize_filename("{subjectName} {sessionName}".format(**lectures[0]))
```
The line length should be less than 88 characters. Follow PEP8 convention.
```suggestion
    subject_name = sanitize_filename(
        "{subjectName} {sessionName}".format(**lectures[0])
    )
```

iamkroot reviewed (changes requested):
Also see [CONTRIBUTING.md](https://github.com/iamkroot/ilc-scraper/blob/master/CONTRIBUTING.md) for guidelines on how to write proper commit messages.

iamkroot commented on pyproject.toml:
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ilc-scraper"
-version = "0.8.2"
+version = "0.8.3"
```
This should not be a part of this PR. I'll bump the scraper version separately. One PR should only handle one change. Consider what would happen if we had to revert this change - we still wouldn't want to rescind the version change for the scraper.

brahma19 replied:
Done.

iamkroot reviewed (changes requested):
After making the necessary changes, please squash all your commits into one, and ensure that the commit message follows the `CONTRIBUTING` guidelines.

brahma19 committed:
Fix target folder name for windows OS
Code derives the folder from the course name from the impartus and when same course is share across multiple tracks e.g. MBA/MTech/PGDBA its name will have
forward slashes in it, windows does not allow folder name with slashes. Using pathvalidate module to clean the folder name that will work on any platform.

Fixes #56

iamkroot reviewed (approved):
LGTM!
